### PR TITLE
feat(about) Check for latest release if user is admin

### LINKED ISSUE #236: Check for new versions
---

Author Name: **Bob Gobeille** 
Original Redmine Issue: 5859, http://www.fossology.org/issues/5859
Original Date: 2013/09/25

---

Add to Help > Check for new version

New UI to check for a new FOSSology version.  If there is a newer version, print out where to get it (package and source) as well as links to the how to install wiki page.

### PR DESCRIPTION
**Pull request by steffen-weber**
fix #236

### CODE REVIEW

File: src/www/ui/page/AboutPage.php [hunk 1]
```diff
@@ -50,15 +50,49 @@ function __construct()
...
         'project' => _("FOSSology"),
-        'copyright' => _("Copyright (C) 2007-2014 Hewlett-Packard Development Company, L.P.<br>\nCopyright (C) 2014-2015 Siemens AG."),
+        'copyright' => _("Copyright (C) 2007-2014 Hewlett-Packard Development Company, L.P.<br>\nCopyright (C) 2014-2016 Siemens AG."),
     );
+    
+    if (\Fossology\Lib\Auth\Auth::isAdmin()) {
+      $latestRelease = $this->curlGit('releases/latest');
+      $since = '?since=' . date('Y-m-dH:i:sZ', time() - 3600 * 24 * 30);
+      $commits = $this->curlGit('commits' . $since);
+      $commit = substr($commits[0]['sha'],0,6);
```
Comment (maximilianhuber) [inline src/www/ui/page/AboutPage.php]: When visiting `http://localhost:8081/repo/?mod=about` with enabled debug information I get the warning
```
Notice: Undefined offset: 0 in /usr/local/share/fossology/www/ui/page/AboutPage.php on line 64
```

--

tmp3

**Comment by maximilianhuber** [comment]
You could also add links to https://github.com/fossology/fossology/ and https://github.com/fossology/fossology/releases/latest (behind the text The latest version 3.0.0)

**Comment by maximilianhuber** [comment]
@steffen-weber: the merged PRs from **shaheemazmalmmd** have introduced merge-conflicts to your PR. Sorry.

**Comment by steffen-weber** [comment]
Got same error when connectivity was bad. So existence of both infos (release/commit) are checked now separately.
-> restructured the feature, added tests, rebased onto master

**Pull request merged**